### PR TITLE
Nook project selector

### PR DESCRIPTION
**Pull request by ktkization**

### CODE REVIEW

File: webapp/lib/controller.dart [hunk 1]
```diff
@@ -99,31 +102,36 @@ void command(UIAction action, Data data) {
 
     /*** Data */
     case UIAction.needsReplyDataUpdated:
-      Map<DateTime, int> data = new Map.fromIterable(needsReplyDataList,
+      view.contentView.projectSelectorView.populateProjects(projectList);
+
+      var selectedProjectNeedsReplyDataList = needsReplyDataList.where((
+        d) => d.project == view.contentView.projectSelectorView.selectedProject).toList();
```
Comment (marianamarasoiu) [inline webapp/lib/controller.dart]: nit: weird line break - can you change it so you have `(d) => ...` and indented 4 spaces from the row above? thanks

File: webapp/lib/view.dart [hunk 2]
```diff
@@ -173,11 +173,41 @@ class BannerView {
...
+      controller.command(controller.UIAction.needsReplyDataUpdated, null);
+    });
+    projectSelector.append(_projectOptions);
+  }
+
+  String get selectedProject => _projectOptions.value;
+
+  void populateProjects(Set<String> options) {
+    for(var option in options) {
+      if(_projectOptions.children.where((opt) => (opt as OptionElement).value == option).length > 0)
```
Comment (marianamarasoiu) [inline webapp/lib/view.dart]: nit: space needed between `for/if` and `(`

File: webapp/web/selector.css [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    -webkit-appearance: none;
+    -moz-appearance: none;
+    -ms-appearance: none;
+    appearance: none;
+    background-color: #f3f3f3;
+    background-repeat: no-repeat;
+    background-image: linear-gradient(45deg, transparent 50%, currentColor 50%), linear-gradient(135deg, currentColor 50%, transparent 50%);
+    background-position: right 15px top 1em, right 10px top 1em;
+    background-size: 5px 5px, 5px 5px;
+  }
```
Comment (marianamarasoiu) [inline webapp/web/selector.css]: nit: needs empty last line, and the closing bracket seems to be indented to 2 spaces rather than 0?

**Review by marianamarasoiu — APPROVED** [review]
LGTM with a few nits, thanks @ktkization !

File: webapp/lib/controller.dart (hunk 1 above)
Comment (ktkization) [reply webapp/lib/controller.dart]: Fixed that

File: webapp/lib/view.dart (hunk 2 above)
Comment (ktkization) [reply webapp/lib/view.dart]: Fixed that

File: webapp/web/selector.css (hunk 3 above)
Comment (ktkization) [reply webapp/web/selector.css]: Fixed that

**Pull request merged**